mandatory/2-logic-error.js

```diff
@@ -1,16 +1,17 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
```

Comment:
It might be a good idea to try and run the tests for this file.
Also, you can see some examples of how to get the length of a string here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length
Compare these examples with your code 😄